src/lib/types/vec3-float32.js

```diff
@@ -0,0 +1,3 @@
+import r from 'restructure';
+
+export default new r.Array(r.floatle, 3);
```

Comment:
Isn't this the same as `float3`? I'd be totally in favour of normalizing all of these to make things more clear 👍